uvicorn/protocols/http/h11_impl.py

```diff
@@ -22,7 +22,7 @@ def _get_status_phrase(status_code):
 
 
 STATUS_PHRASES = {
-    status_code: _get_status_phrase(status_code) for status_code in range(100, 600)
+    status_code: _get_status_phrase(status_code) for status_code in range(200, 599)
```

Comment:
100 to 200 is okay, but you should not change 600 to 599. 599 means `200-598`.

Reply:
ok